apps/graphql/src/types/output/Sector.js

```diff
@@ -15,5 +15,9 @@ export default new GraphQLObjectType({
     duration: { type: GraphQLInt },
     origin: { type: Location },
     segments: { type: new GraphQLList(Segment) },
+    stopoverDuration: {
+      type: GraphQLInt,
+      resolve: ({ stopoverDuration }: any): number | null => stopoverDuration,
```

Comment:
I expect this is of the type `Sector` defined in the `ItinerariesloaderTypes.js`, please import the type from there, and replace any with `Sector`

Reply:
Fixed